docs/INSTALL.md

```diff
@@ -9,23 +9,27 @@ General ZAC usage instructions may be found in the [README.md](../README.md) fil
 
 - Java JDK 17 and up (we use the `Temurin` distribution)
 
+For mac users you can run the following command to install the required Java version:
+
+```shell
+brew install oracle-jdk17
+```
+
 ### Gradle build
 
 The software is built using Gradle and for the final step using Maven.
 Both a Gradle and a Maven wrapper are included in the source code, so you do not need to install either Gradle or Maven yourself.
 To build the software use the following command:
 
 ```shell
-./gradlew build
+
 ```
 
 This builds all the software, including the Java backend as well as the TypeScript frontend (using `npm`), runs all unit tests
 and packages the built software first into a WAR archive and then finally by invoking a Maven command from Gradle into a
 [WildFly application server](https://www.wildfly.org/) bootable fat-JAR. This last step uses [Galleon](https://docs.wildfly.org/galleon/).
 
-All generated artifacts by Gradle are placed into the `build` folder while the final WildFly bootable JAR is placed in the
-`target` folder. The reason why we use Maven for this last step is because there is unfortunately no Gradle alternative for the
-[WildFly Maven Plugin](https://docs.wildfly.org/wildfly-maven-plugin).
+All generated artifacts by Gradle are placed into the `build` folder while the final WildFly bootable JAR is placed in the `target` folder. The reason why we use Maven for this last step is because there is unfortunately no Gradle alternative for the [WildFly Maven Plugin](https://docs.wildfly.org/wildfly-maven-plugin).
```

Comment:
Is this a Markdown formatting thing again? Maybe you could look into how we can all share the same formatting conventions?

Reply:
Maybe we should add some markdown conventions to `.editorconfig`

Reply:
prettier was enabled in my editor, but this project does not have a prettier configuration